spec/bgs/services/vet_record_spec.rb

```diff
@@ -31,12 +31,12 @@
       expect(response[:birth_year]).to eq('54')
       expect(response[:date_of_birth]).to eq('09/08/1954')
       expect(response[:date_of_birth]).to eq('09/08/1954')
-      expect(response[:death_month]).to be_nil
-      expect(response[:death_day]).to be_nil
-      expect(response[:death_century]).to be_nil
-      expect(response[:death_year]).to be_nil
-      expect(response[:date_of_death]).to be_nil
-      expect(response[:pow_number_of_days]).to be_nil
+      expect(response[:death_month]).to eq('03')
+      expect(response[:death_day]).to eq('28')
+      expect(response[:death_century]).to eq('20')
+      expect(response[:death_year]).to eq('21')
+      expect(response[:date_of_death]).to eq('03/28/2021')
+      expect(response[:pow_number_of_days]).to eq('0')
```

Comment:
dang

Reply:
I'm pretty sure these are just test users, but that one kinda surprised me, as well